src/app/dashboard/[workspaceId]/billing/page.tsx

```
@@ -1,22 +1,73 @@
+
+type PaymentData = {
+  data?: {
+    subscription?: {
+      plan: 'PRO' | 'FREE'
+      payments: PaymentHistoryItem[]
+    }
+  }
+  status: number
+}
+
 type Props = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)